### ISSUE TITLE
Uncaught TypeError: Cannot read property 'filter' of undefined

### ISSUE THREAD

**Issue by ahmed-sigmalux** [nierdz/flush-opcache#3]
I'm seeing the below error on the OPcache statistics page
```
settings.php?page=flush-opcache&tab=statistics:8444
Uncaught TypeError: Cannot read property 'filter' of undefined
    at HTMLInputElement.change (settings.php?page=flush-opcache&tab=statistics:8444)
    at HTMLInputElement.u [as __onchange] (d3.v3.min.js:1)
  | change | @ | settings.php?page=flush-opcache&tab=statistics:8444
  | u | @ | d3.v3.min.js:1
```

**Comment by nierdz** [comment]
Yes, this is a bug from [opcache-status](https://github.com/rlerdorf/opcache-status)
It's related to this [PR](https://github.com/rlerdorf/opcache-status/pull/42)
In the next WP Opcache release I'll use another project to provide statistics. Maybe this one : [opcache-status](https://github.com/wp-cloud/opcache-status)

**Comment by nierdz** [comment]
This issue is resolved in v2.3 release. 118f24add01a36c1522bd0ca8bc0647e2cd7912c
